### PR TITLE
Maintain Empty Lines Around Export Statements

### PR DESCRIPTION
**Pull request by OmarJaroudi**

### CODE REVIEW

File: docs/rules/empty-line-around-export.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Rule Details
+
+This rule aims to maintain a convention regarding empty lines before or between "export" statements
+
+Examples of **incorrect** code for this rule:
+
+```js
+
+export const FIRST_CONST = 'first_contsant';
```
Comment (AliGTLogic) [inline docs/rules/empty-line-around-export.md]: s/contsant/constant

File: docs/rules/empty-line-around-export.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+export default class DefaultExportClass {}
+
+```
+
+Examples of **correct** code for this rule:
+
+```js
+
+export const FIRST_CONST = 'first_contsant';
```
Comment (AliGTLogic) [inline docs/rules/empty-line-around-export.md]: s/contsant/constant

**Review by AliGTLogic — CHANGES_REQUESTED** [review]
Nicee

**Review by AliGTLogic — APPROVED** [review]
Every time you make a typo, the errorists win. (googled this)

**Review by khaledsayyed — APPROVED** [review]

**Pull request merged**